Index: libedit/configure.ac
===================================================================
--- libedit.orig/configure.ac	2014-02-19 12:56:18.789024620 +0100
+++ libedit/configure.ac	2014-02-19 12:56:41.309023539 +0100
@@ -35,7 +35,7 @@
 AC_PROG_LIBTOOL
 
 # libtool -version-info
-AC_SUBST(LT_VERSION, [0:48:0])
+AC_SUBST(LT_VERSION, [2:48:0])
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
