Index: libedit.git/configure.ac
===================================================================
--- libedit.git.orig/configure.ac	2013-07-07 12:52:19.153286118 +0200
+++ libedit.git/configure.ac	2013-07-07 13:21:58.109275673 +0200
@@ -31,7 +31,7 @@
 AC_PROG_LIBTOOL
 
 # libtool -version-info
-AC_SUBST(LT_VERSION, [0:45:0])
+AC_SUBST(LT_VERSION, [2:45:0])
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
